batchUploadResourceCollections.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This bash script is used to batch upload resource collections to the iHub server. It iterates through 
+
+# Assumes that you are in the root of the community directory and that it's named like /ihub-community-<COMMUNITY_NAME>
+community=$(basename "$(pwd)" | grep -o '[^-]*$') 
+
+cd ./src/main/flowResources || exit
+
+for dir in ./*; do
+    if [ -d "$dir" ]; then
+        resource=$(basename $dir)
+        echo ""
+        echo "Uploading collection $resource"
+        uploadResourceCollection.sh "$resource" "$community"
+        echo ""
+    fi
+done
+
+echo "Resource collections uploaded successfully!"
```

Comment:
Even if the upload fails, it reports that the upload succeeded:

```
Uploading collection one
no available flow token.  You must have a /Users/tim/creds/two.token file populated with an active flow access token from the server.
rm: one.zip: No such file or directory
  adding: one/ (stored 0%)
/Users/tim/src/developer-scripts/uploadResourceCollection.sh: line 25: return: can only `return' from a function or sourced script
/Users/tim/src/developer-scripts/uploadResourceCollection.sh: line 30: [: !=: unary operator expected
cat: uploadResourceCollectionResponse.txt: No such file or directory

Resource collections uploaded successfully!
```

This is because `uploadResourceCollection.sh` does not return an error exit code.  Feel free to ignore this comment or handle it in a way you see fit.

Reply:
Thanks for calling this out! Let me know what you think about the changes to address this. 

(Admittedly, there's some quality-of-life work to be done here; I haven't seen the   uploadResourceCollectionResponse.txt read successfully to date)

Reply:
You're welcome!